bpf/bpf_host.c

```diff
@@ -1086,6 +1086,15 @@ int to_host(struct __ctx_buff *ctx)
 		traced = true;
 	}
 
+#ifdef ENABLE_IPSEC
```

Comment:
nit, the commit message contradicts this but I think guarding at least by encryption seems slightly safer.

Not sure I fully understand the implications here but this is the "to_host" prog section anyway so clearly if we've configured the system to route traffic towards a device with this prog attached then at this point the traffic must be destined for the prog so setting `PACKET_HOST` seems OK.

Reply:
I can fix description or fix patch. Not sure which is better.

Reply:
:shrug: I'd probably just drop the second paragraph in the commit message. Having some gating around this logic so we at least have one reference point around why this logic needs to exist seems better than having no context.

Reply:
We will backport this patch as well so seems like having the ifdef there is a good idea.